src/engines/kotor/gui/widgets/kotorwidget.cpp

```diff
@@ -365,6 +365,14 @@ void KotORWidget::setText(const Common::UString &text) {
 	_text->setText(text);
 }
 
+void KotORWidget::setHorizontalTextAlign(float halign) {
+	_text->setHorizontalAlign(halign);
```

Comment:
You should wrap these in `if (_text)`.

And yes, all those other places here that access _text (and _quad) should check them as well, like setText() and setFont(). That slipped me by, it seems.

Feel free to do these with commits in this PR or a new PR, otherwise I'll do that once this PR is merged.